### PR TITLE
merge master from dev

### PR DESCRIPTION
**Pull request by ali-hamud**
- implementation of rebind and clone commands.

### CODE REVIEW

File: lib/cli-create-embed-token.ts [hunk 1]
```diff
@@ -16,13 +16,17 @@ export default function CliCreateEmbedToken() {
...
+        .option('-d, --dataset <datasetId>', 'The Power BI dataset id')
+        .option('-s, --scopes [scopes]', 'The permission scopes allowed')
         .option('-u, --username [username]', 'The RLS username')
         .option('--roles [roles]', 'The RLS roles', list);
 
     program.on('--help', function () {
         console.log('  Examples:');
         console.log('');
         console.log('    $ powerbi create-embed-token -c <collection>, -w <workspace> -r <reportId> -k <accessKey> -u [username] --roles [roles1,roles2,...]');
+        console.log('    $ powerbi create-embed-token -c <collection>, -w <workspace> -d <datasetId> -k <accessKey>');
```
Comment (ArinaHantsis) [inline lib/cli-create-embed-token.ts]: We should have added scopes for the example to create dataset token as even the basic loading operation requires Dataset.Read permissions

File: lib/cli-update-connection.ts [hunk 2]
```diff
@@ -21,7 +21,7 @@ export default function CliUpdateConnection() {
     program.on('--help', function () {
         console.log('  Examples:');
         console.log('');
-        console.log('    $ powerbi update-connection -c <collection> -k <accessKey> -w <workspace> -u [username] -p [password] -s [connectionString]');
+        console.log('    $ powerbi update-connection -c <collection> -k <accessKey> -w <workspace> -d <datasetId> -u [username] -p [password] -s [connectionString]');
```
Comment (ArinaHantsis) [inline lib/cli-update-connection.ts]: Why do we change this call?

Comment (ArinaHantsis) [inline lib/cli-update-connection.ts]: Shouldn't it be datasource? Let's check the REST call params first

Comment (ali-hamud) [inline lib/cli-update-connection.ts]: it's datasetId not data source.
was missing and accepted from contributor.

File: lib/cli-create-embed-token.ts (hunk 1 above)
Comment (ali-hamud) [inline lib/cli-create-embed-token.ts]: I will add to examples file. another file. in another submit.

**Pull request merged**